### PR TITLE
Documentation V1

### PR DESCRIPTION
**Pull request by chynu**
First pass at usage instructions.

---
This change is [Reviewable](https://reviewable.io/reviews/google/blockly-devtools/263)

### CODE REVIEW

**Comment by chynu** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[docs/README.md, line 14 at r1](https://reviewable.io:443/reviews/google/blockly-devtools/263#-KrI5x14Vg6XDPjNbWUB:-KrI5x14Vg6XDPjNbWUC:b-d8qvnn) ([raw file](https://github.com/google/blockly-devtools/blob/d3df3195715da8456aed3dfc3d5091e2abddb5f8/docs/README.md#L14)):*
> ```Markdown
> * Build workspace contents (blocks that are loaded onto a Blockly workspace on
>   load)
> * Configure workspace inject options (currently a web-only feature).
> ```

Wondering if workspace configs are indeed a "web-only feature"? I'm just mimicking the old documentation here in this line.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/google/blockly-devtools/263)*

**Comment by AnmAtAnm** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[docs/README.md, line 14 at r1](https://reviewable.io:443/reviews/google/blockly-devtools/263#-KrI5x14Vg6XDPjNbWUB:-KrIJuelVem2LSAVvgaj:b7o2aew) ([raw file](https://github.com/google/blockly-devtools/blob/d3df3195715da8456aed3dfc3d5091e2abddb5f8/docs/README.md#L14)):*
<details><summary><i>Previously, chynu (celine choo) wrote…</i></summary><blockquote>

Wondering if workspace configs are indeed a "web-only feature"? I'm just mimicking the old documentation here in this line.
</blockquote></details>

Yep. Configuration is very different on the other platforms.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/google/blockly-devtools/263)*

File: INSTRUCTIONS.md [hunk 1]
```diff
@@ -50,53 +53,69 @@ through the navigation tree at the left side of the application. Clicking on
...
 A block library is essentially a grouping of block definitions. This makes it
 easier for you, the developer, to organize your blocks to refer to them later,
 save them, and move them around together (as the block definitions that belong
 in one library will be in the same file).
 
-The developer tools application, when opening an empty project, will auto-create
-a default block library, "MyFirstBlockLibrary." You can always add more block
-definitions by using the menubar of the app or by using the add button at the
-bottom left side of the window.
+A well-designed block libraries, should be easy to understand at a high level.
```
Comment (rachel-fenichel) [inline INSTRUCTIONS.md]: libraries -> library

**Comment by rachel-fenichel** [comment]
One nit then lgtm.

**Comment by AnmAtAnm** [comment]
Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions, some commit checks broke.

---

*[INSTRUCTIONS.md, line 73 at r2](https://reviewable.io:443/reviews/google/blockly-devtools/263#-Ku6Vnlth8Y2eiQE5L1z-r2-73:-Ku6qi6VFixHc7vlo1X-:b-896fix) ([raw file](https://github.com/google/blockly-devtools/blob/2642365f6808ab4b9c5c71c36f5bc67a6f2a3805/INSTRUCTIONS.md#L73)):*
<details><summary><i>Previously, rachel-fenichel (Rachel Fenichel) wrote…</i></summary><blockquote>

libraries -> library
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/google/blockly-devtools/263)*

**Pull request merged**